recipes/cad1/task1/baseline/enhance.py

```diff
@@ -0,0 +1,316 @@
+""" Run the dummy enhancement. """
+import json
+import logging
+from pathlib import Path
+from typing import Dict, List, Tuple, Union
+
+import hydra
+import numpy as np
+import pandas as pd
+import torch
+from omegaconf import DictConfig
+from scipy.io import wavfile
+from torchaudio.pipelines import HDEMUCS_HIGH_MUSDB
+from torchaudio.transforms import Fade
+from tqdm import tqdm
+
+from clarity.enhancer.compressor import Compressor
+from clarity.enhancer.nalr import NALR
+
+logger = logging.getLogger(__name__)
+
+
+def separate_sources(
+    model: torch.nn.Module,
+    mix: torch.Tensor,
+    sample_rate: int,
+    segment: float,
+    overlap: float,
+    device: Union[torch.device, str]
```

Comment:
Missing comma is causing `black` to fail on `pre-commit.ci` pipeline...

```suggestion
    device: Union[torch.device, str],
```